doc/source/howtos/upgrade_guide.rst

```diff
@@ -117,3 +120,17 @@ To upgrade the cluster, follow these steps for each node:
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
+.. note:: It is important that when rejoining, the node should synchronize using
+   :term:`IST`. If for some reason :term:`SST` syncronization is used
+   instead, there is a problem, as the node will use data-directory from
```

Comment:
Can you re-word to avoid highlighting this as a problem?
Instead, we need to indicate that if node rejoins please follow the extra step to ensure that the data-directory of the JOINER is compatible with the installed binaries.

Reply:
Idem. Always try and use positive language and to state things in a way that is positive. Avoid things like "throw a lot of errors" "wrong structure", "SST-related problem", etc.

For example;
"When performing any upgrade (major or minor), SST could be initiated by the JOINER node after the upgrade if the server was offline for some time. After SST completes, the data directory structure needs to be upgraded (using mysql_upgrade) once more to ensure compatibility with the higher version DONOR. For this, ..."

Reply:
Ok, that's why this is an "attempt to fix DOC-686" :)
There is a question about mentioning errors. I think we need to specify some indicator - a clear sign that additional steps are necessary. As far as I got into the topic, messages about SST appear in both variants (SST and IST)... they are different but still may confuse... so currently the most obvious indicator I know about is "a lot of errors". Is there any specific sign we can pay user's attention on?

Reply:
Also, it seems like there is still a possibility, that node would join with SST for the second time. I've searched for instruction, which makes node for sure join with IST, and got one, but it it can spoil the whole howto (low-level things done by hands). So changing the phrase "ensure cluster joins back using IST" with some clear indicator (and, may be, external link?) sounds reasonable.

Reply:
1. lot of error is not a good sign. Especially with the minor upgrade. A user can look out for the messages log file that clearly indicates IST or SST. If needed I can pull out those messages

For SST:

2018-02-06T09:14:48.958792Z 2 [Note] WSREP: Check if state gap can be serviced using IST
2018-02-06T09:14:48.958804Z 2 [Note] WSREP: Local UUID: 00000000-0000-0000-0000-000000000000 != Group UUID: 2b1d7ac7-0b1e-11e8-bd3a-e3f0e97b7f5f
2018-02-06T09:14:48.958857Z 2 [Note] WSREP: State gap can't be serviced using IST. Switching to SST

For IST:

2018-02-06T09:16:57.581259Z 2 [Note] WSREP: Receiving IST: 9120 writesets, seqnos 72-9192
2018-02-06T09:16:57.581345Z 0 [Note] WSREP: Receiving IST...  0.0% (   0/9120 events) complete.